draft/spec/index.html

```diff
@@ -1353,9 +1353,10 @@ <h2>BagIt in an OCFL Object</h2>
         <p>
             [[BagIt]] is a common file packaging specification, but unlike the OCFL it does not provide a mechanism for
             content versioning. Using the OCFL it is possible to store a BagIt structure with content versioning, such
-            that when the object state is resolved, it creates a valid BagIt 'bag'. This example will illustrate one way
-            this can be accomplished, using the <a href="https://tools.ietf.org/html/draft-kunze-bagit-17#section-4.1">
-            example of a basic bag</a> given in the BagIt specification.
+            that when the <a>logical state</a> is resolved, it creates a valid BagIt 'bag'. This example will
+            illustrate one way this can be accomplished, using the
+            <a href="https://tools.ietf.org/html/draft-kunze-bagit-17#section-4.1"> example of a basic bag</a> given in
```

Comment:
Should this be https://datatracker.ietf.org/doc/html/rfc8493#section-4.1 ?

Reply:
Thanks Peter!